workshop/examples/emr-eks/ebs-pvc/karpenter-compute-provisioner-ebs/execute_emr_eks_job.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# NOTE: Make sure to set the region before running the shell script e.g., export AWS_REGION="<your-region>"
+
+read -p "Enter the AWS Region: " AWS_REGION
```

Comment:
Why ask for region is `AWS_REGION` is set?

Reply:
Region is used in the script below